pom.xml

```diff
@@ -17,7 +17,7 @@
     <version>1.7.0-SNAPSHOT</version>
 
     <properties>
-        <asyncHttpClientVersion>1.14-MULE-021</asyncHttpClientVersion>
+        <asyncHttpClientVersion>1.14-MULE-022-SNAPSHOT</asyncHttpClientVersion>
```

Comment:
Leave this to the latest release which is 1.14-MULE-022. This probably only works locally.